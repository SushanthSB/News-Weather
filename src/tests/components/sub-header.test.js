import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { spy } from 'sinon';
import SubHeader from '../../components/sub-header'

describe('SubHeader()',() => {
	const props = {
		news_source:['google-news', 'the-hindu']
	}
	it('should render without fail',() => {
		const wrapper = shallow(<SubHeader {...props} />)
		expect(wrapper).toHaveLength(1)
	})
	it('should not get modified',() => {
		const tree = renderer.create(<SubHeader {...props} />).toJSON();
		expect(tree).toMatchSnapshot();
	})
})