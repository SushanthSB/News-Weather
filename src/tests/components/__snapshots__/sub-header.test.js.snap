// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader() should not get modified 1`] = `
<div
  className="jumbotron sub-header col-xs-12 col-sm-12 col-md-12 col-lg-12"
>
  <p>
     Get live headlines from 
  </p>
  <h1>
    <div
      className=""
    />
  </h1>
</div>
`;
